src/main/java/smb/sdk/sample/main.java

```diff
@@ -120,11 +113,11 @@ public static void main( String[] args )
 
             try
             {
-                anfAccount = await(createANFAccount(anfClient, resourceGroupName, anfAccountName, newAccount));
+                anfAccount = createANFAccount(manager.serviceClient(), resourceGroupName, anfAccountName, newAccount);
             }
-            catch (CloudException e)
+            catch (Exception e)
```

Comment:
>Exception [](http://example.com/codeflow?start=19&length=9)

CloudException got replaced by AzureError (at least in Python), and should not be replaced by the broader Exception, can you please apply the equivalent for Java? Please do the same on all samples.